src/platform/user/profile/vet360/tests/e2e/addressValidation.e2e.spec.js

```diff
@@ -0,0 +1,65 @@
+const E2eHelpers = require('platform/testing/e2e/helpers');
+const Timeouts = require('platform/testing/e2e/timeouts');
+const createMockEndpoint = require('platform/testing/e2e/mock-helpers');
+const Auth = require('platform/testing/e2e/auth');
+const routes = require('./routes.json');
+
+function openModal(browser) {
+  const fieldWrapper = `[data-field-name="mailingAddress"]`;
+  const editButton = `${fieldWrapper} [data-action="edit"]`;
+  const updateButton = 'button[data-action="save-edit"]';
+  const addressValidationModal = 'div[id="address-validation-warning"]';
+
+  browser
+    .waitForElementVisible(editButton, Timeouts.verySlow)
+    .click(editButton)
+    .waitForElementVisible(updateButton, Timeouts.verySlow)
+    .click(updateButton)
+    .waitForElementVisible(addressValidationModal, Timeouts.verySlow);
+}
+
+function verifyLabels(browser) {
+  const userEnteredAddressInput = `label[for="userEntered"]`;
+  const firstSuggestedAddressInput = `label[for="0"]`;
+  browser
+    .waitForElementVisible(userEnteredAddressInput, Timeouts.verySlow)
+    .assert.containsText(userEnteredAddressInput, '1493 Martin Luther King Rd')
+    .waitForElementVisible(firstSuggestedAddressInput, Timeouts.verySlow)
+    .assert.containsText(
+      firstSuggestedAddressInput,
+      // eslint-disable-next-line no-useless-concat
+      '400 NW 65th St\n' + 'Seattle, WA 98117',
+    );
+}
+
+function beginTests(browser) {
+  openModal(browser);
+  verifyLabels(browser);
+}
+
+function createMockRoutes(token) {
+  const promises = [];
+  for (const route of routes) {
+    promises.push(createMockEndpoint(token, route));
+  }
+  return Promise.all(promises);
+}
```

Comment:
I think a `.map()` would be more appropriate than a for loop.

```js
function createMockRoutes(routes, token) {
  return Promise.all(
    routes.map(route => createMockEndpoint(token, route)),
  )
}
```

Reply:
truuuuuuue... but this was basically boilerplate e2e test code :-p

Reply:
@bkjohnson I absolutely love the fact that you give us thoughtful PR reviews and comments (seriously, 100%) but in a case like this when the only comment is really a nit, would it be possible to also give an approval so that we can make the suggested change and then merge? Would speed things up and make things a bit easier for all of us, IMO.

Reply:
Updated!

Reply:
Ahh sorry, I got sidetracked by something else.  Approving now.